package.json

```diff
@@ -21,11 +23,36 @@
   },
   "homepage": "https://github.com/vasinkevych/react-native-node#readme",
   "dependencies": {
+    "axios": "^0.15.3",
+    "babel-core": "^6.17.0",
+    "babel-loader": "^6.2.0",
+    "babel-plugin-add-module-exports": "^0.1.2",
+    "babel-plugin-react-html-attrs": "^2.0.0",
+    "babel-plugin-transform-class-properties": "^6.3.13",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "bootstrap-css-only": "^4.1.1",
+    "classnames": "^2.2.5",
     "dotenv": "^6.2.0",
     "koa": "^2.0.0-alpha.8",
+    "koa-static": "^5.0.0",
     "koa2-router": "^1.0.6",
     "nodemailer": "^5.1.1",
-    "pm2": "^3.2.9"
+    "pm2": "^3.2.9",
+    "react": "^0.14.6",
```

Comment:
0.14.6  -> https://reactjs.org/versions

Reply:
lets use latest version for all packages (e.g. react, react-redux, react-router)